scripts/ci

```diff
@@ -7,4 +7,3 @@ cd $(dirname $0)
 ./test
 ./validate
 ./validate-ci
-./package
```

Comment:
Any reason ./package is removed? If anything is wrong on Dockerfile the build won't be able to catch.

Reply:
On the side note, I think we should add local copy for development per darren's concern. Today if I want to build image locally the curl won't work. And also the ./package should just run local copy for build step so that CI can catch `./package` script.

Reply:
This needs to be re-added; good catch

Reply:
Re-added